docker/ci/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM ruby:2.7.1-buster
+MAINTAINER operations@openproject.com
+
+ENV USER=dev
+RUN useradd -d /home/$USER -m $USER
+WORKDIR /home/$USER
+
+RUN wget --quiet -O- https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt buster-pgdg main" > /etc/apt/sources.list.d/pgdg.list
+
+RUN apt-get update -qq && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    postgresql-9.6 postgresql-client-9.6 time pandoc default-jre-headless
+
+ENV CI=true
+ENV RAILS_ENV=test
+ENV NODE_VERSION="12.18.3"
+ENV BUNDLER_VERSION="2.1.4"
+ENV BUNDLE_WITHOUT="development:production:docker"
+ENV DATABASE_URL=postgres://app:p4ssw0rd@127.0.0.1/app
+ENV CHROME_SOURCE_URL=https://dl.google.com/dl/linux/direct/google-chrome-stable_current_amd64.deb
+ENV JOBS=4
+
+RUN wget --no-verbose -O /tmp/$(basename $CHROME_SOURCE_URL) $CHROME_SOURCE_URL && \
+  apt install -y /tmp/$(basename $CHROME_SOURCE_URL) && rm -f /tmp/$(basename $CHROME_SOURCE_URL)
+
+RUN curl -s https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-x64.tar.gz | tar xzf - -C /usr/local --strip-components=1
+RUN gem install bundler --version "$BUNDLER_VERSION" --no-document
+
+COPY ./entrypoint.sh /usr/sbin/entrypoint.sh
+
+VOLUME [ "/usr/local/bundle", "/home/$USER/openproject", "/home/$USER/openproject/tmp" ]
+
+WORKDIR /home/$USER/openproject
+
+ENTRYPOINT [ "/usr/sbin/entrypoint.sh" ]
+
+EXPOSE 3000
```

Comment:
Is this necessary? we could set it in the docker command or the compose file anyway, and even expose 3312 if we wanted to

Reply:
I didn't find a way to dynamically expose `CI_JOBS` ports from the docker-compose file. Hence that (ugly but limited to the CI dockerfile) solution. An alternative would be to generate an addition Compose file to override it, but that seemed overkill.

Reply:
Now that I think about it, dynamically appending a list of EXPOSE instructions at the end of this Dockerfile would also work fine.